.gitignore

```diff
@@ -97,6 +97,7 @@ themes/*/cache/*
 
 # Config
 
+config/defines_custom.inc.php
```

Comment:
What's the relation with this Docker PR?

Reply:
this file is created while running the container or in some situation https://github.com/PrestaShop/PrestaShop/blob/develop/config/config.inc.php#L30
but didn't know why it wasn't be ignored :thinking: